src/main/groovy/no/skatteetaten/aurora/gradle/plugins/JavaApplicationTools.groovy

```diff
@@ -176,48 +176,57 @@ class JavaApplicationTools {
         ])
   }
 
-  AuroraReport applySpring(String starterVersion, Boolean devTools) {
-
+  AuroraReport applySpring(String starterVersion, Boolean devTools, Boolean bootJarEnabled) {
     def implementationDependencies = [
         "com.fasterxml.jackson.datatype:jackson-datatype-jsr310",
         "no.skatteetaten.aurora.springboot:aurora-spring-boot2-starter:$starterVersion",
     ]
     if (devTools) {
       implementationDependencies.add("org.springframework.boot:spring-boot-devtools")
     }
+
     log.info("Apply Spring support")
-    project.with {
 
+    project.with {
       apply plugin: 'io.spring.dependency-management'
 
-      [jar, distZip]*.enabled = true
-      [bootJar, distTar, bootDistTar, bootDistZip]*.enabled = false
+      if (!bootJarEnabled) {
+        [jar, distZip]*.enabled = true
+        [bootJar, distTar, bootDistTar, bootDistZip]*.enabled = false
 
-      configurations.archives.artifacts.removeIf {
-        if (it.hasProperty("archiveTask")) {
-          !it.archiveTask.enabled
-        } else {
-          !it.delegate.archiveTask.enabled
+        configurations.archives.artifacts.removeIf {
+          if (it.hasProperty("archiveTask")) {
+            !it.archiveTask.enabled
+          } else {
+            !it.delegate.archiveTask.enabled
+          }
         }
       }
 
       springBoot {
         buildInfo()
       }
-      dependencies {
 
+      dependencies {
         implementationDependencies.each { implementation it }
       }
+    }
 
+    def resolvedBootJarText = {
+      if (bootJarEnabled) {
+        "."
+      } else {
+        ", bootJar disabled."
+      }
     }
```

Comment:
```suggestion
    def resolvedBootJarText = bootJarEnabled ? "" : ", bootJar disabled" // set period in description-string
```

Reply:
Good point, blind to ternaries after too long with Kotlin